### PR TITLE
next/194/20231119/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
 - #9756
 - #9819
 - #9830

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 16632

File: doc/userguide/rules/file-keywords.rst [hunk 1]
```diff
@@ -5,6 +5,36 @@ Suricata comes with several rule keywords to match on various file
 properties. They depend on properly configured
 :doc:`../file-extraction/file-extraction`.
 
+file.data
+---------
+
+The ``file.data`` sticky buffer matches on contents of files that are
```
Comment (inashivb) [inline doc/userguide/rules/file-keywords.rst:11]: nit: trailing whitspace.

**Review by inashivb — COMMENTED** [review]
One nit inline, rest looks good.

**Review by inashivb — APPROVED** [review]
The nit came from the original PR. Nothing changed in this one. Since that was approved, setting this to approved too.

**Pull request merged**